Content.Shared/Blocking/Components/BlockingComponent.cs

```diff
@@ -25,16 +25,18 @@ public sealed partial class BlockingComponent : Component
     [ViewVariables, AutoNetworkedField]
     public bool IsBlocking;
 
+    // imp - removed this functionality
     /// <summary>
     /// The ID for the fixture that's dynamically created when blocking
     /// </summary>
```

Comment:
rider is throwing warnings about xaml comments being improperly placed, it's honestly probably worth deleting both of these entirely since they're unused